### PR TITLE
Add /place/stat-vars API (replacement for /place/stats-var)

### PR DESCRIPTION
**Pull request by shifucun**
Once all clients are migrated over, can remove "/place/stats-var".

Tested in Minikube and looking fine

### CODE REVIEW

**Review by beets — COMMENTED** [review]
Is it possible to use the new request/response protos for the older API so we don't have to duplicate the handler code (and protos)?

File: proto/mixer.proto [hunk 1]
```diff
@@ -449,6 +449,24 @@ message GetPlaceStatsVarResponse {
   map<string, StatsVars> places = 1;
 }
 
+
+// StatVars represent a list of statistical variable dcids.
+message StatVars {
+  repeated string statVars = 1;
```
Comment (pradh) [inline proto/mixer.proto:455]: snake case or camel case?

**Review by pradh — APPROVED** [review]

**Comment by shifucun** [comment]
> Is it possible to use the new request/response protos for the older API so we don't have to duplicate the handler code (and protos)?

Consolidated the implementation code. The response can not be reused since the json field name changed from "statsVars" to "statVars".

File: proto/mixer.proto (hunk 1 above)
Comment (shifucun) [reply proto/mixer.proto]: Done

**Review by pradh — APPROVED** [review]

**Review by beets — APPROVED** [review]
thanks!

**Comment by shifucun** [comment]
Thanks for review!

**Pull request merged**